ci/launch_cluster.sh

```diff
@@ -13,13 +17,24 @@ CHANNEL="$1"
 VARIANT="$2"
 
 if [ "$VARIANT" == "open" ]; then
-  TEMPLATE="https://s3.amazonaws.com/downloads.dcos.io/dcos/${CHANNEL}/cloudformation/multi-master.cloudformation.json"
+  INSTALLER_URL="https://downloads.dcos.io/dcos/${CHANNEL}/dcos_generate_config.sh"
```

Comment:
I wonder if we could ge pre-baked AWS images at some point.

Reply:
That would be the best, but this means that we are releasing new DC/OS images on every release.

Reply:
That'd be fine IMHO. Anyhow, that's out of scope.